### ISSUE TITLE
copy edit

### ISSUE THREAD

**Issue by MissPhilbin** [raspberrypilearning/turtley-amazing#1]
Hi @lauraclay & @LornaLynch - When you have time, would you mind checking the copy of this resource. 

For the first time we've embedded a python editor into the resource. I'd be really interested in the outcome of someone testing all the steps work with the editor. If one of you would be willing to act as a learner (it is a beginner resource, no previous knowledge required) it would be a huge help. To do this you will need to look at the resource on [staging](https://staging.raspberrypi.org/learning/turtley-amazing/worksheet/) and not github as the editor uses iframes. 

Thanks Carrie Anne.

**Comment by LornaLynch** [comment]
On it now.

**Comment by LornaLynch** [comment]
@MissPhilbin The Trinket examples in the first worksheet look great, but I'm getting a single-coloured in the first example on the second worksheet, rather than a multi-coloured one.

**Comment by MissPhilbin** [comment]
@LornaLynch Thanks, we are just looking at that example now. Good spot!

**Comment by LornaLynch** [comment]
Also, this may be me being a bear of very little brain, but in the Modulo section of worksheet 2 (the Trinket example), is there any way we could add a line to clarify exactly what is happening in `turtle.color(colours[i % len(colours)])`. I was putting myself in the position of having to explain it to a group, and I couldn't figure out how I would do it.

**Comment by LornaLynch** [comment]
I'm also having problems with the enormous lists example: I can get it to work if there's only one colour value being changed, but trying to change all three means I go out of the top and bottom of the range (see image). Again, this may be me, but if I am making a really dim, obvious error, it might be worth flagging it up as a possible pitfall? "Remember to check/change foo", or that sort of thing. 
![screen shot 2016-04-06 at 11 09 16](https://cloud.githubusercontent.com/assets/11164126/14313213/32d28aae-fbe8-11e5-8748-66b8ee5c0b69.png)

**Comment by LornaLynch** [comment]
Other than the queries above, everything looks good.

**Comment by LornaLynch** [comment]
Another issue I've run into: I tried to replicate the code in my desktop Python editor: I copied and pasted it straight into IDLE, and I keep getting error messages (attached). I'm not sure if there is a really obvious reason why this is happening, but I wanted to flag it up in case users try to C&P and have similar results. I'm not really knowledgeable enough to see what I have done wrong, sorry.... 

If it's me being dense, please give me a virtual slap. 


![screen shot 2016-04-06 at 14 20 59](https://cloud.githubusercontent.com/assets/11164126/14320936/f8613448-fc0e-11e5-87bf-5a8843e28ff3.png)

**Comment by MissPhilbin** [comment]
@LornaLynch The last issue is down to versions of python. Unfortunately trinket is python 2, whereas typically we use python 3 in our resources. It is highly unlikely that anyone will copy and paste this into IDLE 3 but we can add a note that it is python 2 syntax. 

@MarcScott Can you look at the other points made above as they fall on worksheet 2. Thanks.

**Comment by LornaLynch** [comment]
I did wonder the version: I got similar error messages in Python 2, but I have 2.7, so perhaps that is what is causing the problem. Thanks for the clarification!

**Comment by MarcScott** [comment]
1. The colours have been fixed to make the turtle more visible.
2. The issue with colours outside the range (0-255) is the use of `>=` and `<=` rather than just `<` and `>`. However, as the colours aren't used at this stage, it doesn't really have much of an effect on the code.
3. I've added a comment into the code to explain the `%` line. This is a more advanced line of code, and it is not essential for it to be understood. It is necessary though to give a pretty spiral at the end :smile:
